js/app.js

```diff
@@ -16,4 +18,108 @@ navLinks.forEach((link) => {
 
     navMenu.classList.remove('active');
   });
-});
\ No newline at end of file
+});
+
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard  ',
+    image: '../images/work-image1.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the1500, when an unknown printer took a galley of type and scrambled it 1960s.',
+    image: '../images/work-image2.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+  {
+    id: 3,
+    name: 'Facebook 360',
+    description:
+      ' Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. scrambled it 1960s with the relea',
+    image: '../images/work-image3.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+  {
+    id: 4,
+    name: 'Uber Navigation',
+    description:
+      'Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley.',
+    image: '../images/work-img4.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+];
+
+const onModalOpen = async (id) => {
+  const getData = await projects.find((project) => project.id === id);
+
+  const modal = document.querySelector('.modal-container');
+  modal.style.display = 'block';
+  document.getElementById('modal-title').innerHTML = getData.name;
+  document.getElementById('modal-desc').innerHTML = getData.description;
+  document.getElementById('modal-img').src = getData.image;
+  document.getElementById('modal-see-live').src = getData.link;
+  document.getElementById('modal-see-source').src = getData.source;
```

Comment:
- [ ] Note that the HTML markup for your projects in the project section need to be loaded dynamically. You do not have to write any HTML. Here you can see that you are trying to access some HTML elements and then populate them with your data. Please delete everything concerning your project in an HTML file. ONLY add an empty container, then all the content will be added dynamically. If you don't get well what you have to do, please refer to the following project pull request and then do your work 💡  👉 [LINK](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-2a099ca3b81cdfe23a8dc8eb7677c597dbb0996f90476b3dd497e0bc36fb0f69)

Reply:
@BertrandConxy Everything works fine from my end and that of my coding partner

Reply:
I have tried this previously and i had issues with the buttons not working. i feel very frustrated and i feel like giving up. This is the instruction that best works at the moment.

Reply:
I have fixed all issues @BertrandConxy